src/components/ContentSidebar/ActivityFeed/activity-feed/__tests__/ActivityFeed-test.js

```diff
@@ -761,4 +766,30 @@ describe('components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed', ()
             );
         });
     });
+
+    describe('addCurrentVersion()', () => {
```

Comment:
Not sure if this test already exists but it would be worth adding a test to ensure that the empty state shows up  when only 1 version item is in feedItems and then that the active state shows up when you have 1 version and anything else

Reply:
I didn't see the test, so added one.